cfgov/jinja2/v1/your-story/index.html

```diff
@@ -0,0 +1,302 @@
+                        public without your permission.
+                        </p>
+                    </div>
+                </div>
+            </div>
+            <div class="content-l_col content-l_col-1">
+                <div class="m-info-unit m-info-unit__inline">
+                    <div class="m-info-unit_image m-info-unit_image__square"
+                    style="background-image: url( {{ static('nemo/_/img/isocons-alt-green-analyze-and-report.png' ) }} );"></div>
+                    <div class="m-info-unit_content">
+                        <h3>3. Your story helps others</h3>
+                        <p>The more we hear from you, the more insight we gain into what’s
+                        happening in the financial world and how it’s affecting you. We see
+                        trends and work to head off problems. <a title="Learn about enforcement actions" href="/policy-compliance/enforcement/actions/">Learn about our recent enforcement actions</a>.
+                        </p>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class="block
+                block__flush-bottom
+                block__padded-bottom">
+        <div>Have a story to share with the CFPB?</div>
+        <a class="a-btn a-btn__full-on-xs u-mt15"
+           href="/your-story">
+            Tell your story
+        </a>
+    </div>
+</div>
+{% endblock %}
+
+{% block content_sidebar -%}
+<div class="consumer-stories">
+    <header class="m-slug-header">
+        <h2 class="a-heading">
+            EVERYONE HAS A STORY
+        </h2>
+    </header>
+    <ul class="m-list
+               m-list__unstyled
+               m-list__links">
+        <li class="m-list_item">
+            <p>
+                Learn how the CFPB has helped others take action
+                to address their financial problems while helping
+                them feel like they were not alone.
+            </p>
+        </li>
+        <li class="m-list_link">
+            <a href="/everyone-has-a-story/leah-repaying-student-loans/"
+               class="consumer-stories_image-container">
+                <img src="{{ static( 'nemo/_/img/Leah_thumbnail.png') }}"
+                     alt="Leah thumbnail">
+            </a>
+        </li>
+        <li class="m-list_link">
+            <a href="/everyone-has-a-story/leah-repaying-student-loans/"
+               class="h4">Leah: Repaying student loans</a>
+        </li>
+        <li class="m-list_item">
+            <p>
+                Leah was struggling to keep up with her student loan debt,
+                until she learned that she had options with the
+                Paying for College Repayment Tool.
+            </p>
+        </li>
+    </ul>
+    <div class="block">
+        <ul class="m-list
+                   m-list__unstyled
+                   m-list__links">
+            <li class="m-list_link">
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/3248/commits/76abd9ebb3ebea57757555b95469fc4569ab938d.